modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -0,0 +1,233 @@
+// Used in cicd/builds/running-entitled-builds.adoc
+
+[id="builds-running-entitled-builds-with-sharedsecret-objects_{context}"]
+= Running entitled builds using SharedSecret objects
+
+[role="_abstract"]
+Configure and perform a build in one namespace that securely uses RHEL entitlements from a `Secret` object in another namespace.
+
+Previously, {product-title} 4.9.x and earlier suggested accessing RHEL entitlements from OpenShift Builds by creating a `Secret` object with your subscription credentials in the same namespace as your `Build` object. Now, with {product-title} 4.10 and later, you can access your credentials and certificates from a `Secret` object in one of the {product-title} system namespaces. You run entitled builds with CSI volume mount of a `SharedSecret` custom resource (CR) instance that references the `Secret` object.
+
+This procedure relies on the newly introduced The Shared Resources CSI Driver feature, which you can use to declare CSI Volume mounts in {product-title} Builds. It also relies on the {product-title} Insights Operator.
+
+:FeatureName: The Shared Resources CSI Driver
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+:FeatureName: TBD
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+The Shared Resources CSI Driver and TBD features also belong to the `TechPreviewNoUpgrade` feature set, which is a subset of the current Technology Preview features. You can enable the `TechPreviewNoUpgrade` feature set on test clusters, where you can fully test them while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents updates. This feature set is not recommended on production clusters. See "Enabling Technology Preview features using feature gates" in the following "Additional resources" section.
```

Comment:
Looking at https://deploy-preview-40993--osdocs.netlify.app/openshift-enterprise/latest/cicd/builds/running-entitled-builds.html#builds-running-entitled-builds-with-sharedsecret-objects_running-entitled-builds I think the two included items above are sufficient, and this extra item here is redundant and should be removed